src/common/config.cc

```diff
@@ -946,83 +952,177 @@ int md_config_t::set_val_impl(const char *val, const config_option *opt)
 
 int md_config_t::set_val_raw(const char *val, const config_option *opt)
 {
+  const int CEPH_SETTING_SAFE = 1;
```

Comment:
@amitkumar50 , that was when I started working on Ceph (that's an old PR) and I used it as a way to identify a constant. However, as it ended up getting old and a rebase was needed, I ended up not touching it (once it is a pretty big change) 'till we were certain this PR would still be useful or not. Should this PR still be interesting, I will rebase and rework whatever is needed.

Thanks,